### PR TITLE
Data Wrangling Assignment -- playing with (and ruining) dplyr

### PR DESCRIPTION
**Pull request by sparrow925**
Won't be my last push before Thursday... but trying to remember where to make pull requests from... This one is from MY fork of env-info

### CODE REVIEW

File: students/sparrow925.Rmd [hunk 1]
```diff
@@ -40,4 +40,53 @@ Creating code that pushes graphs reflecting up-to-date datasets without doing mo
...
+# get count per country
+d3=aggregate(city_name ~ country_code, data=d2, FUN='length')
+# write out csv
+write.csv(d3, './data/sparrow925_EUSO2_citycount.csv', row.names = FALSE)
+```
+...There is a better way to format this...
+
+
+# read in csv
+#... be careful, there is something wrong here and it crashes R
```
Comment (bbest) [inline students/sparrow925.Rmd:83]: crashing eh? will try this out on mine

**Comment by bbest** [comment]
yeah, feel free to make pull requests in b/n :+1:, especially if you have a question and want me to look at your code

**Comment by bbest** [comment]
fyi, no crashing on my Mac running that last chunk

**Comment by sparrow925** [comment]
Hmmmm I'll fuss with it. Thanks for running it!

**Comment by sparrow925** [comment]
The problem doesn't seem to be the code itself, which runs well and then creates the final table... however, R crashes when I try to view that table that it has created. Any ideas?

**Pull request merged**